starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,6 +1,78 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="main.css">  
+  <title>Apple Pie</title>
+  <!-- Google fonts gives us access to the Open Sans Font -->
+  <!-- Google fonts is also awesome! Make sure to check it out -->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+</head>
+<body>
+   
+    
+      <div class= "crop">
+        <img src="apple-pie.jpg">
+      </div>
+      <h1>Apple Pie</h1>
+      <p>
```

Comment:
You can wrap all the content (except for the heading) into a div and apply some margin there.